src/main/java/io/github/cottonmc/resources/CommonResources.java

```diff
@@ -23,20 +23,21 @@ public static void initialize() {
         builtinMetal("copper", BlockSuppliers.STONE_TIER_ORE, MACHINE_AFFIXES);
         builtinMetal("silver", BlockSuppliers.IRON_TIER_ORE, MACHINE_AFFIXES);
         builtinMetal("lead", BlockSuppliers.IRON_TIER_ORE, MACHINE_AFFIXES);
-        builtinMetal("zinc", BlockSuppliers.STONE_TIER_ORE);
-        builtinMetal("aluminum", BlockSuppliers.IRON_TIER_ORE);
-        builtinMetal("cobalt", BlockSuppliers.IRON_TIER_ORE);
-        builtinMetal("tin", BlockSuppliers.STONE_TIER_ORE);
-        builtinMetal("tungsten", BlockSuppliers.IRON_TIER_ORE);
-
-        builtinMetal("platinum", BlockSuppliers.IRON_TIER_ORE);
-        builtinMetal("palladium", BlockSuppliers.IRON_TIER_ORE);
-        builtinMetal("osmium", BlockSuppliers.IRON_TIER_ORE);
-        builtinMetal("iridium", BlockSuppliers.IRON_TIER_ORE);
+        builtinMetal("zinc", BlockSuppliers.STONE_TIER_ORE, MACHINE_AFFIXES);
+        builtinMetal("aluminum", BlockSuppliers.IRON_TIER_ORE, MACHINE_AFFIXES);
+        builtinMetal("cobalt", BlockSuppliers.IRON_TIER_ORE, MACHINE_AFFIXES);
+        builtinMetal("tin", BlockSuppliers.STONE_TIER_ORE, MACHINE_AFFIXES);
+        builtinMetal("titanium", BlockSuppliers.IRON_TIER_ORE, MACHINE_AFFIXES);
+        builtinMetal("tungsten", BlockSuppliers.IRON_TIER_ORE, MACHINE_AFFIXES);
+
+        builtinMetal("platinum", BlockSuppliers.IRON_TIER_ORE, MACHINE_AFFIXES);
+        builtinMetal("palladium", BlockSuppliers.IRON_TIER_ORE, MACHINE_AFFIXES);
+        builtinMetal("osmium", BlockSuppliers.IRON_TIER_ORE, MACHINE_AFFIXES);
+        builtinMetal("iridium", BlockSuppliers.IRON_TIER_ORE, MACHINE_AFFIXES);
 
         builtinMetal("steel", null, MACHINE_AFFIXES);
         builtinMetal("brass", null, MACHINE_AFFIXES);
-        builtinMetal("electrum", null); //no ore, no gears/plates
+        builtinMetal("electrum", null, MACHINE_AFFIXES); //no ore, no gears/plates
```

Comment:
The comment here says no gears/plates, which directly contradicts this change. Which side do we wanna take?

Reply:
was told to generate json for everything even if it does not make sense, I think the idea is that we will just not have them in by default later, but as of right now with the cascade failure issues was told to register everything